go.mod

```diff
@@ -87,6 +87,7 @@ require (
 	github.com/sirupsen/logrus v1.8.1 // indirect
 	github.com/tsaarni/x500dn v0.0.0-20210331182804-14283c7f5a16 // indirect
 	github.com/xlab/treeprint v1.1.0 // indirect
+	go.opentelemetry.io/proto/otlp v0.19.0 // indirect
```

Comment:
why is this added ?

Reply:
because of `github.com/envoyproxy/go-control-plane/envoy/extensions/access_loggers/open_telemetry/v3`

Reply:
ah thanks for the clarification !